### DISCUSSION TITLE
Is Auth automatically backed up?

### DISCUSSION THREAD

**Discussion by vinoff** [ente-io/ente#1965] [Q&A]
I see no info on the app whether or not my TOTP keys are backed up. Authy has that info.

If I lose my phone or something to the same effect and re-install Ente-auth using my e-mail/password, will my codes be there?

**Comment by ua741** [comment]
If you are signed in, the app will automatically try to back up the codes. For codes that are not backed up, you will see an unsynced icon. The app will attempt to back them up whenever there is an internet connection available.
